packaging/macosx/Info.plist.in

```diff
@@ -18,5 +18,6 @@
 	<string>6.0</string>
 	<key>CFBundleVersion</key>
 	<string>@SHORT_VERSION@</string>
+	<key>NSSupportsAutomaticGraphicsSwitching</key><true/>
```

Comment:
the commit title and commit content seem to disagree. I think you are turning automatic GPU switching **on**, not **off**